pilot/pkg/serviceregistry/kube/controller/pod.go

```diff
@@ -104,14 +104,14 @@ func (pc *PodCache) event(obj interface{}, ev model.Event) error {
 				// add to cache if the pod is running or pending
 				pc.keys[ip] = key
 				if pc.c != nil && pc.c.XDSUpdater != nil {
-					pc.c.XDSUpdater.WorkloadUpdate(ip, pod.ObjectMeta.Labels, pod.ObjectMeta.Annotations)
+					pc.c.XDSUpdater.WorkloadUpdate(pc.c.ClusterID, ip, pod.ObjectMeta.Labels, pod.ObjectMeta.Annotations)
```

Comment:
does kubernetes have this field ClusterID ? and is it guaranteed to be unique? or is pod cache our own struct?